dotnet/Capstone/DAO/BirdNoteSqlDao.cs

```diff
@@ -0,0 +1,187 @@
+            {
+                throw new DaoException("SQL exception occurred", ex);
+            }
+        }
+
+        public BirdNote getBirdSighting(int id)
+        {
+            BirdNote birdSighting = null;
+            string sql = "SELECT id, bird_id, date_sighted, males_spotted, females_spotted, feeder_type, food_blend, notes FROM bird_sighting WHERE id = @id";
+
+            try
+            {
+                using (SqlConnection conn = new SqlConnection(connectionString))
+                {
+                    conn.Open();
+
+                    SqlCommand cmd = new SqlCommand(sql, conn);
+                    cmd.Parameters.AddWithValue("@id", id);
+                    SqlDataReader reader = cmd.ExecuteReader();
+
+                    if (reader.Read())
+                    {
+                        birdSighting = MapRowToBirdSighting(reader);
+                    }
+                }
+            }
+            catch (SqlException ex)
+            {
+                throw new DaoException("SQL exception occurred", ex);
+            }
+
+            return birdSighting;
+        }
+
+        public List<BirdNote> getSightings(int birdId)
+        {
+            List<BirdNote> sightingList = new List<BirdNote>();
+
+            string sql = "SELECT id, bird_id, date_sighted, males_spotted, females_spotted, feeder_type, food_blend, notes FROM bird_sighting WHERE bird_id = @bird_id";
+
+            try
+            {
+                using (SqlConnection conn = new SqlConnection(connectionString))
+                {
+                    conn.Open();
+
+                    SqlCommand cmd = new SqlCommand(sql, conn);
+                    cmd.Parameters.AddWithValue("@bird_id", birdId);
+                    SqlDataReader reader = cmd.ExecuteReader();
+
+                    while (reader.Read())
+                    {
+                        BirdNote birdSighting = MapRowToBirdSighting(reader);
+                        sightingList.Add(birdSighting);
+                    }
+                }
+            }
+            catch (SqlException ex)
+            {
+                throw new DaoException("SQL exception occurred", ex);
+            }
+
+            return sightingList;
+        }
+
+        private BirdNote MapRowToBirdSighting(SqlDataReader reader)
+        {
+            BirdNote birdSighting = new BirdNote();
+            birdSighting.NoteId = Convert.ToInt32(reader["id"]);
+            birdSighting.BirdId = Convert.ToInt32(reader["bird_id"]);
+            birdSighting.DateSpotted = Convert.ToDateTime(reader[("date_sighted")]);
+            birdSighting.NumMales = Convert.ToInt32(reader["males_spotted"]);
+            birdSighting.NumFemales = Convert.ToInt32(reader["females_spotted"]);
+            birdSighting.FeederType = Convert.ToString(reader["feeder_type"]);
+            birdSighting.FoodBlend = Convert.ToString(reader["food_blend"]);
+            birdSighting.Notes = Convert.ToString(reader["notes"]);
```

Comment:
Because these items are not marked "NOT NULL" in the schema, there's a chance for them to cause errors when pulling a null value. See the [GetSafeString](https://github.com/ahdisease/Bird-Finder/blob/654e710664ed8688628956d2bd7b0a53f0878d21/dotnet/Capstone/DAO/UserProfileSqlDao.cs#L238) method in UserProfileSqlDao for an example of how to handle this.